### PR TITLE
Update changelog for 0.18.0.

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,23 +1,43 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.17.8: August 16, 2018
-* Fix attach to process for systems without `bash` by using `sh` instead. [#569](https://github.com/Microsoft/vscode-cpptools/issues/569)
-  * Andy Neff (@andyneff) [PR 2340](https://github.com/Microsoft/vscode-cpptools/pull/2340)
-* Fix IntelliSense crash after hover or completion with `_Complex` types. [#689](https://github.com/Microsoft/vscode-cpptools/issues/689), [#1112](https://github.com/Microsoft/vscode-cpptools/issues/1112)
+## Version 0.18.0: August 16, 2018
+### New Features
+* Add `C_Cpp.disableLanguageService` setting (for users who only want the debugger features). [#785](https://github.com/Microsoft/vscode-cpptools/issues/785)
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: this will need to be updated based on the changes requested to the active PR, but you can update it in that PR.

**Review by bobbrow — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
Yeah, don't merge this yet -- some changes might get cut for 0.18.0.

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: It's updated.

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
+* Fix `Cpp.clang_format_path` not accepting environment variables. [#2344](https://github.com/Microsoft/vscode-cpptools/issues/2344)
+* Fix IntelliSense not working with non-ASCII characters in the WSL install path. [#2351](https://github.com/Microsoft/vscode-cpptools/issues/2351)
 * Filter out buggy IntelliSense error `"= delete" can only appear on the first declaration of a function`. [#2352](https://github.com/Microsoft/vscode-cpptools/issues/2352)
+* Fix IntelliSense failing with WSL if gcc is installed bug g++ isn't. [#2360](https://github.com/Microsoft/vscode-cpptools/issues/2360)
+* Fix WSL paths starting with `/mnt/` failing to get symbols parsed. [#2361](https://github.com/Microsoft/vscode-cpptools/issues/2361)
+* Fix IntelliSense process crash when hovering over a designated initializer list with an anonymous struct. [#2370](https://github.com/Microsoft/vscode-cpptools/issues/2370)
+* Stop showing "File: " in completion details for internal compiler defines. [#2387](https://github.com/Microsoft/vscode-cpptools/issues/2387)
```
Comment (pieandcakes) [inline Extension/CHANGELOG.md:38]: * Fix to allow SIGINT to be sent using the `kill -2` command when using `pipeTransport`.

**Pull request merged**